modules/tensor_mechanics/tests/Wave_1_D/HHT_time_integration/wave_bc_1d.i

```diff
@@ -0,0 +1,346 @@
+# Wave propogation in 1-D using HHT time integration
+#
+# The test is for an 1-D bar element of length 4m  fixed on one end
+# with a sinusoidal pulse dirichlet boundary condition applied to the other end.
+# alpha, beta and gamma are Newmark  time integration parameters
+# The equation of motion in terms of matrices is:
+#
+# M*accel + K*((1+alpha)*disp-alpha*disp_old) = 0
+#
+# Here M is the mass matrix, K is the stiffness matrix
+#
+# The displacement at the second, third and fourth node at t = 0.1 are
+# -8.097405701570538350e-02, 2.113131879547342634e-02 and -5.182787688751439893e-03, respectively.
+
+[GlobalParams]
+  order = FIRST
```

Comment:
FIRST, LAGRANGE are the default values anyways, so it should be possible to omit the GlobalParams block entirely.